clowder-framework/clowder2
# 506 frontend request refresh jwt token every minute

This PR implement a periodically request to update refresh token on any of the private route every minute. 

-------------
To test:
1. watch the console --> network tab see if it's refreshing every minute
2. change the interval in `app.config.js` to a very long time; then wait 5 minutes to try clicking anything on frontend to test the refresh original pattern
